contributing/README.md

```diff
@@ -85,6 +85,39 @@ the time to ensure your contributions are high quality and easy for our
 community to review and accept. Please don't hesitate to [reach out to
 us][Slack] if you have any questions about contributing!
 
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of Origin
+(DCO). This document was created by the Linux Kernel community and is a simple
+statement that you, as a contributor, have the legal right to make the
+contribution. See the [DCO](../DCO) file for details.
+
+Contributors sign-off that they adhere to these requirements by adding a
+Signed-off-by line to commit messages. For example:
+
+```
+This is my commit message
+
+Signed-off-by: Random J Developer <random@developer.example.org>
+```
+
+Git even has a -s command line option to append this automatically to your
+commit message:
+
+```
+$ git commit -s -m 'This is my commit message'
+```
+
+If you have already made a commit and forgot to include the sign-off, you can
+amend your last commit to add the sign-off with the following command, which can
+then be force pushed.
+
+```
+git commit --amend -s
+```
+
+We use a [DCO bot] to enforce the DCO on each pull request and branch commits.
```

Comment:
Just curious — what is a “branch commit” in this context? Just a commit to any branch?

Reply:
Ha I have no idea - i think just saying "each pull request" is more accurate and clear.  I'll make that update now 😊

Reply:
I changed it to "to enforce the DCO on all commits in every pull request" - thanks for them good eyes @hasheddan! 👀